dasharo-security/verified-boot.robot

```diff
@@ -63,12 +63,30 @@ VBO007.002 Boot from RW when correctly signed firmware is flashed
     ${out_vboot}=    Execute Command In Terminal    ./dasharo-tools/vboot/workbuf_parse -1 | grep "boot mode"
     Should Contain    ${out_vboot}    Normal boot mode
 
+VBO008.001 Booting from recovery
+    [Documentation]    Check whether the information about recovery mode will be
+    ...    displayed after flash firmware with wrong vboot keys. The boot should
+    ...    continue automatically after a 30s delay.
+    Skip If    not ${VERIFIED_BOOT_POPUP_SUPPORT}    VBO008.002 not supported
+    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    VBO008.002 not supported
+    Boot System Or From Connected Disk    ubuntu
+    Boot System Or From Connected Disk    ubuntu
```

Comment:
this line is duplicated, is that intentional?

Reply:
Yes, I can't imagine why would it need to be duplicated. Must have missed it because when connection method is SSH this KW simply does nothing.

https://github.com/Dasharo/open-source-firmware-validation/compare/b605efb7ca9d230d2e74ddc82728a647aaa06157..1c41ca0469a70a368fe1d8219c427fcc83246c9c#diff-b36e38045b372731533f567c2d8f5bc80dc8666f034271da81bdb51483074df9L73